### PR TITLE
Add ability to show celery history by using redis and celery events

### PR DESCRIPTION
**Pull request by riltsken**
Also adds tests for the playbook

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.PHONY: clean test test_default dependencies
+
+all: test
+
+clean:
+	vagrant destroy -f
+
+dependencies:
+	ansible-galaxy install -f jdauphant.nginx,v1.3.4
+	ansible-galaxy install -f zenoamaro.supervisord,v0.3.1
```
Comment (riltsken) [inline Makefile:10]: Not sure how to do this cleanly

File: templates/gunicorn_start.j2 [hunk 2]
```diff
@@ -11,7 +11,7 @@ exec gunicorn \
     --workers {{ crosshair_gunicorn_workers }} \
     --timeout {{ crosshair_gunicorn_timeout }} \
     --user gunicorn \
-    --group crosshair \
+    --group gunicorn \
```
Comment (riltsken) [inline templates/gunicorn_start.j2:14]: Getting permission issues here. No reason to use crosshair for this I don't think?

File: Makefile (hunk 1 above)
Comment (AMeng) [inline Makefile:10]: We can make a `requirements.yml` file?

File: handlers/main.yml [hunk 3]
```diff
@@ -6,3 +6,7 @@
 - name: restart celery
   supervisorctl: name=celery state=present
   supervisorctl: name=celery state=restarted
+
+- name: restart celery events
```
Comment (AMeng) [inline handlers/main.yml]: If we're always restarting both celery processes, we could just combine them into a single notify.

Comment (riltsken) [inline handlers/main.yml]: done

**Comment by AMeng** [comment]
:+1:

**Pull request merged**